WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
```

Comment:
One thing that came to my mind (I couldn't find the part of the code where this is implemented, so I am adding this as a feedback to time limits here).

Some months ago, we had a discussion about whether cumulative and "normal" time limits can exist together or not (see [here](https://github.com/thewca/wca-regulations/issues/374)).

If I am not completely misunderstanding something, the key/value pair for time limits here is: `"cumulative": true|false`, which excludes the option to have both a cumulative and a "normal" time limit.

I'm personally fine with this, but this would actually be a decision for the WRC issue I linked above.